deploy/helm/sumologic/upgrade-1.0.0.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+MAN="Thank you for upgrading to v1.0.0 of the Sumo Logic Kubernetes Collection Helm chart.
+As part of this major release, the format of the values.yaml file has changed.
+
+This script will automatically take the configurations of your existing values.yaml
+and return one that is compatible with v1.0.0.
+
+Usage:
+  ./upgrade-1.0.0.sh /path/to/values.yaml
+
+Returns:
+  new.yaml
+
+For more details, please refer to Migration steps and Changelog here: [link]
```

Comment:
todo: add a proper link once changelog is ready

Reply:
Are we going to add info about requirements (`yq`) in this file or in `Migration steps`

Reply:
I was thinking we could add that to the migration doc - but we could add that here as well